itest/rust/src/object_tests/property_template_test.rs

```diff
@@ -107,6 +111,9 @@ fn property_template_test(ctx: &TestContext) {
           }*/
     }
 
+    // Freeing this early to avoid leaks
+    rust_properties.free();
```

Comment:
```suggestion
    rust_properties.free();
```
I think it's fine without a comment. It's not _that_ important to have no leaks if tests fail, the OS will clean up. Many unit tests are susceptible to this.